pkg/webui/components/status/status.styl

```diff
@@ -0,0 +1,73 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.container
+  display: flex
+  align-items: center
+
+.status
+  position: relative
+  display: inline-block
+  width: 5px
+  height: 5px
+  border-radius: 50%
+  transition: background-color $ad.m
+
+  &:after
+    content: ''
+    position: absolute
+    border-radius: 50%
+    width: 5px
+    height: 5px
+    opacity: .7
+    animation: pulse 2s infinite
+
+  &-good
+    background-color: $c-active-blue
+
+    &:after
+      animation: goodPulse 2s infinite
```

Comment:
The animation is actually thought to pulse whenever an event arrives. It can then also be a little more noticeable. Obviously not MVP critical, so don't do this.